packages/app/src/sections/dashboard/Stake/EscrowTable.tsx

```diff
@@ -182,47 +172,45 @@ const EscrowTable = () => {
 					columnsDeps={columnsDeps}
 					columns={[
 						{
-							Header: () => <input type="checkbox" checked={checkAllState} onChange={selectAll} />,
-							Cell: (cellProps: CellProps<EscrowData>) => (
+							header: () => <input type="checkbox" checked={checkAllState} onChange={selectAll} />,
+							cell: (cellProps) => (
 								<input
 									key={cellProps.row.index}
 									type="checkbox"
 									checked={checkedState[cellProps.row.index]}
 									onChange={handleOnChange(cellProps.row.index)}
 								/>
 							),
-							accessor: 'selected',
-							width: 40,
+							accessorKey: 'selected',
+							size: 40,
 						},
 						{
-							Header: () => (
+							header: () => (
 								<TableHeader $small>{t('dashboard.stake.tabs.escrow.amount')}</TableHeader>
 							),
-							Cell: (cellProps: CellProps<EscrowData>) => (
+							cell: (cellProps) => (
 								<TableCell>{truncateNumbers(cellProps.row.original.amount, 4)}</TableCell>
 							),
-							accessor: 'amount',
-							width: 80,
+							accessorKey: 'amount',
+							size: 80,
 						},
 						{
-							Header: () => (
+							header: () => (
 								<TableHeader $small>{t('dashboard.stake.tabs.escrow.early-vest-fee')}</TableHeader>
 							),
-							Cell: (cellProps: CellProps<EscrowData>) => (
```

Comment:
I take it removing these types all over we lose the type checking now within the row render?

Reply:
Making the table generic means we don't.